test/tools/unified-spec-runner/match.ts

```diff
@@ -523,6 +564,51 @@ export function matchesEvents(
   }
 }
 
+export function compareLogs(
+  expected: ExpectedLogMessage[],
+  actual: ExpectedLogMessage[],
+  entities: EntitiesMap
+): void {
+  expect(actual).to.have.lengthOf(expected.length);
+
+  for (const [index, actualLog] of actual.entries()) {
```

Comment:
We still need the index to set the root prefix appropriately, so I think it's better to keep it as it is since we'd just have to add an index variable either way